frontend/src/components/ShootWorkers/ManageWorkers.vue

```diff
@@ -110,15 +116,56 @@ export default {
     allZones () {
       return this.zonesByCloudProfileNameAndRegion({ cloudProfileName: this.cloudProfileName, region: this.region })
     },
+    currentZonesWithNetworkConfigInShoot () {
+      return map(this.currentZonesNetworkConfiguration, 'name')
+    },
+    currentFreeNetworks () {
+      if (this.isNewCluster) {
+        return this.allZones.length
+      }
```

Comment:
what would happen if you call `findFreeNetworks(this.currentZonesNetworkConfiguration, this.existingWorkerCIDR, this.cloudProviderKind, this.allZones.length)` instead of `this.allZones.length`? Will it return the same value?

Reply:
in this case `findFreeNetworks` would run into an error as `currentZonesNetworkConfiguration` is undefined. However, to simplify code, we could enhance the method to return all networks in case `currentZonesNetworkConfiguration` is undefined. WDYT?

Reply:
yes sounds good

Reply:
done